buildlib/azure-pipelines-pr.yml

```diff
@@ -47,6 +47,32 @@ stages:
                  echo "##vso[task.complete result=Failed;]"
               fi
             condition: eq(variables['Build.Reason'], 'PullRequest')
+  - stage: Format
+    jobs:
+      # Check that the code is formatted according to the code style guidelines
+      - job: format
+        displayName: format code
+        container: fedora
+        steps:
+          - checkout: self
+            clean: true
+
+          - bash: |
+              echo "Checking code format on diff remotes/origin/${SYSTEM_PULLREQUEST_TARGETBRANCH}..${BUILD_SOURCEVERSION}"
+              git-clang-format --diff remotes/origin/${SYSTEM_PULLREQUEST_TARGETBRANCH} ${BUILD_SOURCEVERSION} > format.patch
+              echo "Generated patch file:"
+              cat format.patch
+              if [ "`cat format.patch`" = "no modified files to format" ]; then
+                exit
+              fi
+              git apply ${AGENT_TEMPDIRECTORY}/format-patch
```

Comment:
isn't is supposed to be format.patch (and not ` ${AGENT_TEMPDIRECTORY}/format-patch`)?

Reply:
I removed it